packages/gradle/src/utils/exec-gradle.ts

```diff
@@ -53,3 +68,35 @@ export function execGradleAsync(
     });
   });
 }
+
+/**
+ * This function recursively finds the nearest gradlew file in the workspace
+ * @param filePath the path to start searching for gradlew file
+ * @returns the relative path of the gradlew file to workspace root, throws an error if gradlew file is not found
+ * It will return gradlew.bat file on windows and gradlew file on other platforms
+ */
+export function findGraldewFile(
+  filePath: string,
```

Comment:
This isn't necessarily a file path, so let's call it path

Reply:
changed